goldmane/proto/api.proto

```diff
@@ -19,7 +19,7 @@ service Flows {
   // Namespaces and source / destination names. It allows progressive filtering of criteria based on
   // other filters. i.e., return the flow destinations given a source namespace.
   // Note that this API provides hints to the UI based on past flows and other values may be valid.
-  rpc FilterHints(FilterHintsRequest) returns (stream FilterHint);
+  rpc FilterHints(FilterHintsRequest) returns (FilterHintsResponse);
```

Comment:
This is a change worth talking about I think.

We've been streaming results as a performance measure - presumably this is so that we can include the total number of items in the response?

Alternatives might include adding a new type of `FilterHint` included at the end of the results - e.g., you get a stream of `FilterHint` objects followed by one that just includes the `Total` value and other metadata.

I bring this up because we are going to need to solve this same problem for Flows, which I think have a much more compelling use-case for returning a `stream FlowResult`

Reply:
We also use `Result` as the suffix for flows, so probably best to stick with that terminology for consistency.

Reply:
Updated. Let me know what you think.